components/raftstore-v2/src/worker/pd/slowness.rs

```diff
@@ -135,8 +135,11 @@ where
             STORE_SLOW_TREND_RESULT_GAUGE.set(slow_trend_result_rate);
             STORE_SLOW_TREND_RESULT_VALUE_GAUGE.set(total_query_num);
         } else {
-            // Just to mark the invalid range on the graphic
+            // It means that raftstore thread already hung on IO operating, report a rate
```

Comment:
The QPS will get a `Some(0)` not a `None`. And `None` means that this is a fake heartbeat used to report the current statistics on `SlowTrend` to PD, which used to let PD percept the slowness of this node.

Reply:
What if user set a small store heartbeat interval (<1s)?

Reply:
And it will get a `None`, and mark it with `-100`. But it won't be a FP case, as the PD will check the Duration trend is slow or not simultaneously. Only if  is `Duration Trend` slow and does `QPS Trend` keep descending, it will be marked as a slow node in PD.

Reply:
Got it. It might be a misleading point to reviewers. I've reverted it.